images/downloader/Dockerfile

```diff
@@ -12,12 +10,16 @@ RUN apt-get update -qq && apt-get install -y -q \
     pip install --upgrade pip && \
     rm -rf /root/.cache/pip*
 
-COPY pkg/workloads/cortex/downloader/requirements.txt /src/cortex/downloader/requirements.txt
-RUN pip install -r /src/cortex/downloader/requirements.txt && \
+COPY pkg/workloads/cortex/serve/cortex_internal.requirements.txt /src/cortex/serve/cortex_internal.requirements.txt
+
```

Comment:
Consider moving setup.py to packages folder

Reply:
Not to be done.